qml/Stage/WindowResizeArea.qml

```diff
@@ -101,7 +101,10 @@ MouseArea {
             state = WindowStateStorage.WindowStateNormal;
         }
         windowStateStorage.saveState(root.windowId, state & ~WindowStateStorage.WindowStateMinimized); // clear the minimized bit when saving
-        windowStateStorage.saveGeometry(root.windowId, Qt.rect(priv.normalX, priv.normalY, priv.normalWidth, priv.normalHeight));
+        if ( priv.normalWidth !== 0 )
```

Comment:
should `normalHeight` also be checked here? as either way we don't them to be stored as 0.

Also this could be shortened to `if (priv.normalWidth) {` as 0 equates to false.